doc/example-keywords.txt

```diff
@@ -18,9 +18,11 @@ fuel cell
 heat transfer
 ignition delay
 internal combustion engine
+jet-stirred reactor
 kinetics
 Matlab
 mixture
+mixture rule
 multicomponent transport
```

Comment:
I don't think this is going to be a well-understood keyword to most users, and I'd recommend just leaving it out.